libcxx/include/fstream

```diff
@@ -936,7 +936,11 @@ basic_filebuf<_CharT, _Traits>::seekoff(off_type __off, ios_base::seekdir __way,
   default:
     return pos_type(off_type(-1));
   }
-#    if !_LIBCPP_HAS_OFF_T_FUNCTIONS
+#    if defined(_LIBCPP_MSVCRT_LIKE)
```

Comment:
I think it would be better if we refactor this to have an internal `fseek` instead of duplicating this multiple times. We also have something very similar in `sync()`, which you don't modify here.

Reply:
Ok, I factorized this into `__fseek` and `__ftell` now and use them in these both locations, and in `sync()` which I had missed.